asf_heat_pump_suitability/pipeline/suitability/calculate_suitability.py

```diff
@@ -190,14 +190,6 @@ def get_enhanced_epc(path) -> pl.DataFrame:
     df = df.filter(
         ~pl.col("UPRN").str.contains("dummy"), pl.col("COUNTRY") != "Scotland"
     )
-
-    df = df.with_columns(
-        pl.when(pl.col("listed_building_grade").is_null())
-        .then(False)
-        .otherwise(True)
-        .alias("listed_building"),
-    )
-
```

Comment:
This conversion is no longer needed because `pipeline/prepare_features/listed_buildings.py` functions have been updated to output a flag with `True`/`False` already for listed building status.

Reply:
**CORRECTION**:
This conversion is no longer needed because `pipeline/run_scripts/run_add_features.py` has been been updated to output a flag with `True`/`False` already for listed building status.